datafusion/expr/src/test/function_stub.rs

```diff
@@ -189,3 +202,73 @@ impl AggregateUDFImpl for Sum {
         AggregateOrderSensitivity::Insensitive
     }
 }
+
+pub struct Count {
```

Comment:
Could we perhaps add some basic documentation, like 

```suggestion
/// Testing stub implementation of COUNT aggregate
pub struct Count {
```